build.gradle

```diff
@@ -0,0 +1,74 @@
+import java.nio.file.Files
+import java.nio.file.Paths
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+apply plugin: "java"
+apply plugin: 'maven'
+apply plugin: "distribution"
+apply plugin: "idea"
+
+group "org.logstash.outputs"
+
+sourceCompatibility = JavaVersion.VERSION_1_8
+
+repositories {
+    mavenCentral()
+}
+
+task wrapper(type: Wrapper) {
+    gradleVersion = '4.0'
+}
+
+dependencies {
+    compile 'org.apache.kafka:kafka-clients:1.0.0'
+    compile 'org.slf4j:slf4j-log4j12:1.7.21'
+    compile 'org.lz4:lz4-java:1.4'
+    compile 'org.xerial.snappy:snappy-java:1.1.4'
+    compile 'org.slf4j:slf4j-api:1.7.25'
+    compile 'log4j:log4j:1.2.17'
```

Comment:
This may be outside of this review ... but it seems that that log4j stuff shouldn't be here at compile scope, maybe testCompile ? 

It seems that this targeting slf4j -> log4j version 1, when it should be log4j2 ?

It's fine to follow up after the fact, if this is just a re-implementation of existing code.

Reply:
The dependencies are the same as for the existing plugin. I can follow up with a later task to correct/review log4j/slf4j versions/